files/setup-011-fluentbit.sh

```diff
@@ -0,0 +1,253 @@
+#!/bin/bash
+# Copyright 2019 VMware, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-2
+
+# Setup FluentBit
+
+set -euo pipefail
+
+# Retrieve the FluentBit image
+VEBA_BOM_FILE=/root/config/veba-bom.json
+FLUENTBIT_IMAGE=$(jq -r < ${VEBA_BOM_FILE} '.["fluentbit"].containers[0].name')
+FLUENTBIT_VERSION=$(jq -r < ${VEBA_BOM_FILE} '.["fluentbit"].containers[0].version')
+
+if [ -n "${SYSLOG_SERVER_HOSTNAME}" ]; then
```

Comment:
I'd recommend pulling this logic out and place it in `setup.sh` following https://github.com/vmware-samples/vcenter-event-broker-appliance/blob/e3c43e2f6109cfd91a7025118255b236af602ee7/files/setup.sh#L110 as an example. This way, if there's no syslog input, we don't even bother running script and then you can get rid of the statement where we skip fluentbit config

Reply:
Love it! I will change it.

Reply:
Changes made as suggested @lamw.